### ISSUE TITLE
Bookmarks 0.7.1 - app cannot be installed - owncloud 9.0 or lower required

### ISSUE THREAD

**Issue by whoshonky** [owncloud/bookmarks#279]
### Steps to reproduce
1.  Updated OwnCloud from 9,0,4 to 9.1
2.  Tried to enable Bookmarks App version 0.7
3. got error
**ERROR:
This app cannot be installed because the following dependencies are not fulfilled:
ownCloud 9.0 or lower is required.**
4. updated Bookmarks App manually to version 0.7.1
5. same error

### Expected behaviour
App would re-enable

### Actual behaviour
Cannot re-enable Bookmarks App
**ERROR:
This app cannot be installed because the following dependencies are not fulfilled:
ownCloud 9.0 or lower is required.**

### Server configuration
**Operating system**:
Debian 7.11

**Web server:**
 Apache/2.2.22 (Debian)

**Database:**
MySQL Ver 14.14 Distrib 5.5.50, for debian-linux-gnu (x86_64) using readline 6.2

**PHP version:**

**ownCloud version:** (see ownCloud admin page)
9.1

**Bookmarks version:** (see ownCloud apps page)
0.7.1

**Updated from an older ownCloud or fresh install:**
Updated from 9.0.4
**Where did you install ownCloud from:**
apt
**Signing status (ownCloud 9.0 and above):**
no idea
```
Login as admin user into your ownCloud and access 
http://example.com/index.php/settings/integrity/failed 
No errors have been found.
```

**List of activated apps:**

```
Enabled:
  - activity: 2.3.2
  - calendar: 1.3.2
  - comments: 0.3.0
  - contacts: 1.3.1.0
  - dav: true
  - documents: true
  - external: 1.2
  - federatedfilesharing: 0.3.0
  - files: 1.5.1
  - files_pdfviewer: 0.8.1
  - files_sharing: 0.10.0
  - files_texteditor: 2.1
  - files_trashbin: 0.9.0
  - files_versions: 1.3.0
  - files_videoplayer: 0.9.8
  - firstrunwizard: 1.1
  - gallery: 15.0.0
  - mail: 0.5.2
  - provisioning_api: 0.5.0
  - systemtags: 0.3.0
  - tasks: 0.9.2
  - templateeditor: 0.1
  - updatenotification: 0.2.1
Disabled:
  - bookmarks
  - encryption
  - federation
  - files_antivirus
  - files_external
  - notifications
  - user_external
  - user_ldap
```

**The content of config/config.php:**

```
{
    "system": {
        "instanceid": "ocad84c0b26f",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "192.168.1.6",
            "MYDOMAIN"
        ],
        "mail_domain": "MYDOMAIN",
        "mail_smtpdebug": false,
        "mail_smtpmode": "smtp",
        "mail_smtphost": "MYDOMAIN",
        "mail_smtpport": "25",
        "mail_smtpauth": 1,
        "mail_smtpauthtype": "LOGIN",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "datadirectory": "\/data\/shared",
        "dbtype": "mysql",
        "version": "9.1.0.15",
        "installed": true,
        "forcessl": true,
        "theme": "",
        "maintenance": false,
        "secret": "***REMOVED SENSITIVE VALUE***",
        "mail_from_address": "system",
        "mail_smtpsecure": "tls",
        "loglevel": 2,
        "trashbin_retention_obligation": "auto",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "updatechecker": false,
        "dbname": "owncloud",
        "dbhost": "localhost",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "debug": true,
        "appstore.experimental.enabled": true
    },
    "apps": {
        "activity": {
            "enabled": "yes",
            "installed_version": "2.3.2",
            "types": "filesystem"
        },
        "backgroundjob": {
            "lastjob": "22272"
        },
        "bookmarks": {
            "enabled": "no",
            "installed_version": "0",
            "ocsid": "168710",
            "types": ""
        },
        "calendar": {
            "enabled": "yes",
            "installed_version": "1.3.2",
            "ocsid": "168707",
            "signed": "true",
            "types": ""
        },
        "comments": {
            "enabled": "yes",
            "installed_version": "0.3.0",
            "types": "logging"
        },
        "contacts": {
            "enabled": "yes",
            "installed_version": "1.3.1.0",
            "ocsid": "168708",
            "types": ""
        },
        "core": {
            "global_cache_gc_lastrun": "1438270008",
            "installedat": "1401021697.1983",
            "lastcron": "1470791736",
            "lastupdateResult": "{\"version\":\"8.2.4\",\"versionstring\":\"ownCloud 8.2.4\",\"url\":\"https:\\\/\\\/download.owncloud.org\\\/community\\\/owncloud-8.2.4.zip\",\"web\":\"https:\\\/\\\/doc.owncloud.org\\\/server\\\/8.2\\\/admin_manual\\\/maintenance\\\/upgrade.html\"}",
            "lastupdatedat": "0",
            "oc.integritycheck.checker": "[]",
            "public_caldav": "calendar\/share.php",
            "public_calendar": "calendar\/share.php",
            "public_documents": "documents\/public.php",
            "public_files": "files_sharing\/public.php",
            "public_gallery": "gallery\/public.php",
            "public_webdav": "files_sharing\/publicwebdav.php",
            "remote_caldav": "dav\/appinfo\/v1\/caldav.php",
            "remote_calendar": "dav\/appinfo\/v1\/caldav.php",
            "remote_carddav": "dav\/appinfo\/v1\/carddav.php",
            "remote_contacts": "dav\/appinfo\/v1\/carddav.php",
            "remote_core.css": "\/core\/minimizer.php",
            "remote_core.js": "\/core\/minimizer.php",
            "remote_dav": "dav\/appinfo\/v2\/remote.php",
            "remote_files": "dav\/appinfo\/v1\/webdav.php",
            "remote_filesync": "files\/appinfo\/filesync.php",
            "remote_webdav": "dav\/appinfo\/v1\/webdav.php",
            "repairlegacystoragesdone": "yes",
            "shareapi_allow_mail_notification": "yes",
            "shareapi_allow_public_notification": "yes",
            "shareapi_default_expire_date": "yes",
            "shareapi_enforce_expire_date": "no",
            "shareapi_enforce_links_password": "yes",
            "umgmt_send_email": "true",
            "umgmt_show_backend": "true",
            "umgmt_show_email": "true"
        },
        "documents": {
            "enabled": "yes",
            "installed_version": "0",
            "ocsid": "168711",
            "types": "",
            "unstable": "false"
        },
        "federatedfilesharing": {
            "enabled": "yes",
            "installed_version": "0.3.0",
            "types": ""
        },
        "files": {
            "backgroundwatcher_previous_file": "164",
            "backgroundwatcher_previous_folder": "429",
            "cronjob_scan_files": "500",
            "default_quota": "1 GB",
            "enabled": "yes",
            "installed_version": "1.5.1",
            "types": "filesystem"
        },
        "files_pdfviewer": {
            "enabled": "yes",
            "installed_version": "0.8.1",
            "ocsid": "166049",
            "types": ""
        },
        "files_sharing": {
            "enabled": "yes",
            "installed_version": "0.10.0",
            "types": "filesystem"
        },
        "files_texteditor": {
            "enabled": "yes",
            "installed_version": "2.1",
            "ocsid": "166051",
            "types": ""
        },
        "files_trashbin": {
            "enabled": "yes",
            "installed_version": "0.9.0",
            "types": "filesystem"
        },
        "files_versions": {
            "enabled": "yes",
            "installed_version": "1.3.0",
            "types": "filesystem"
        },
        "files_videoplayer": {
            "enabled": "yes",
            "installed_version": "0.9.8",
            "types": ""
        },
        "files_videoviewer": {
            "enabled": "no",
            "installed_version": "0.1.3",
            "ocsid": "166054",
            "types": ""
        },
        "firstrunwizard": {
            "enabled": "yes",
            "installed_version": "1.1",
            "ocsid": "166055",
            "types": ""
        },
        "gallery": {
            "enabled": "yes",
            "installed_version": "15.0.0",
            "types": ""
        },
        "mail": {
            "enabled": "yes",
            "installed_version": "0.5.2",
            "ocsid": "169914",
            "signed": "true",
            "types": ""
        },
        "provisioning_api": {
            "enabled": "yes",
            "installed_version": "0.5.0",
            "types": "prevent_group_restriction"
        },
        "search_lucene": {
            "enabled": "no",
            "installed_version": "0.5.3",
            "ocsid": "168709",
            "types": "filesystem"
        },
        "systemtags": {
            "enabled": "yes",
            "installed_version": "0.3.0",
            "types": "logging"
        },
        "tasks": {
            "enabled": "yes",
            "installed_version": "0.9.2",
            "ocsid": "164356",
            "signed": "true",
            "types": ""
        },
        "templateeditor": {
            "enabled": "yes",
            "installed_version": "0.1",
            "types": ""
        },
        "updatenotification": {
            "enabled": "yes",
            "installed_version": "0.2.1",
            "types": ""
        },
        "updater": {
            "enabled": "no",
            "installed_version": "0.6",
            "types": ""
        }
    }
}

```

**Are you using external storage, if yes which one:** local/smb/sftp/...
No
**Are you using encryption:** yes/no
No
**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
No

### Client configuration
**Browser:**

**Operating system:**

#### Web server error log
```
Insert your webserver log here
```

#### ownCloud log (data/owncloud.log)
```
I don't have a log at 'data/owncloud.log'.
/var/log/owncloud.log is empty.
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log 
c) ...
```

**Comment by blizzz** [comment]
Yes, there's no release supporting OC 9.1

**Comment by whoshonky** [comment]
Will there be a release supporting OwnCloud 9.1?

**Comment by blizzz** [comment]
I might manage to prepare something for with the Nextcloud 10 release, which most likely is compatible with Oc 9.1. Depends on how much time i can find.

**Comment by tflidd** [comment]
The github-version here is already marked to be compatible up to OC 9.2. Are there any substantial changes or is it just about testing the app with the new release? If it is only testing, a user could take the current master-version from github or change the version dependencies:
https://github.com/owncloud/bookmarks/blob/master/appinfo/info.xml#L18 and report back.

**Comment by blizzz** [comment]
master is on 9.2, yes, and stable9.1 on 9.1. I run my master branch against Nextcloud master and cannot tell how it behaves against ownCloud. However I am not aware of any incompatibilities here or there. So  it's about testing.

**Comment by blizzz** [comment]
#280 is a bug in master and stable9.1 branch for instance

**Comment by blizzz** [comment]
I published a 0.9.0 release for OC & Nc >= 9 last weekend.

**Comment by whoshonky** [comment]
Awesome! Thanks a lot @blizzz 

**Please** keep the updates coming for OwnCloud!!
